test/integration/user_can_view_a_past_order_test.rb

```diff
@@ -0,0 +1,61 @@
+require "test_helper"
+
+class UserCanViewAPastOrder < ActionDispatch::IntegrationTest
+  # Background: An existing user that has one previous order
+  #     As an authenticated user
+  #     When I visit "/orders"
+  #     Then I should see my past order
+  #     And I should see a link to view that order
+  #     And when I click that link
+  #     Then I should see each item that was ordered with the quantity and line-item subtotals
+  #     And I should see links to each item's show page
+  #     And I should see the total price for the order
+
+  #     And I should see the current status of the order (paid, completed, cancelled)
+  #     And I should see the date/time that the order was submitted
+  #     If the order was completed or cancelled
+  #     Then I should see a timestamp when the action took place
+  #     And if any of the items in the order were retired (no longer able to be purchased)
+  #     Then they should still be able to access the item page
+  #     But they should not be able to add the item to their cart
+
+  test 'user can view a past order' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.